xml/dhcp_vendor_class.xml

```diff
@@ -518,6 +518,15 @@
     <param pos="0" name="os.vendor" value="Cisco"/>
   </fingerprint>
 
+  <fingerprint pattern="^Cisco Telepresence Peripheral$">
```

Comment:
Is this a particular device or a family of devices?

Reply:
Agree, family is more suitable, changed.